src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -635,6 +644,51 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		return xterm;
 	}
 
+	private _getXtermTheme(theme?: IColorTheme): ITheme {
```

Comment:
I'd prefer this stayed in xtermTerminal, could we track target in TerminalInstance and pass it through to XtermTerminal in the ctor instead?

Reply:
i'll try that

Reply:
just looked into moving the tests that you have for this over to terminalInstance.test.ts and would be a pain, so this is a better idea